ScreenRuler/Forms/HelpForm.cs

```diff
@@ -15,8 +15,15 @@ public HelpForm()
         {
             InitializeComponent();
             this.KeyPreview = true;
-            // Set lang code
+            // Set lang and check if there is a help file for it.
+            // If not, notify user and fall back to English.
             lang = CultureInfo.CurrentUICulture.TwoLetterISOLanguageName;
+            if (!File.Exists(HelpFileLocation))
+            {
+                var languageName = CultureInfo.CurrentUICulture.DisplayName;
+                MessageBox.Show($"No help file found for {languageName}. Showing English help.", "Language-specific help file not found");
```

Comment:
I think we don't need this extra message box here. If this fallback happens, the rest of the UI usually will be in English already anyway.

Reply:
OK! This was my husband's suggestion and I was also thinking of a situation where a help file hasn't yet been translated (even if the UI was), but then again, the help file is so small that it's effortless to translate along with everything else.